cmd/vulcan-vulners/main.go

```diff
@@ -265,20 +278,28 @@ func analyzeReport(target string, nmapReport *gonmap.NmapRun) ([]report.Vulnerab
 					continue
 				}
 				summary := fmt.Sprintf(vulnersVuln.Summary, port.Service.Product)
-				v, ok := uniqueVulns[summary]
+				uniqueVulnId := fmt.Sprintf("CPE: %s, Port: %d/%s", string(cpe), port.PortId, port.Protocol)
+				var cves []string
+
+				for _, row := range f.Resources.Rows {
+					cves = append(cves, row["CVE"])
+				}
+				v, ok := uniqueVulns[uniqueVulnId]
 				if !ok {
 					v.Vuln = report.Vulnerability{
-						Summary:         summary,
-						Description:     fmt.Sprintf(vulnersVuln.Description, port.Service.Product),
-						Recommendations: vulnersVuln.Recommendations,
+						ID:               computeVulnerabilityID(cves),
+						Summary:          summary,
+						Description:      fmt.Sprintf(vulnersVuln.Description, port.Service.Product),
+						Recommendations:  vulnersVuln.Recommendations,
+						AffectedResource: fmt.Sprintf("%s, %d/%s", string(cpe), port.PortId, port.Protocol),
```

Comment:
Oh, I also forgot to mention in the review that we should add also the `Labels` attribute to the finding, specifying at least the type of finding. In the case of `vulcan-vulners` we would be using the label `issue` for all the findings. So it could be something like:
```
Labels: []string{"issue"},
```